client/dashboard/sites/hosting-feature-activation-modal/data-center-form.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, Card, CardBody, SelectControl } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import InlineSupportLink from '../../components/inline-support-link';
+import { Text } from '../../components/text';
+
+const DataCenterOptions = [
+	{
+		value: '',
+		get label(): string {
+			return __( 'Optimal data center' );
+		},
+	},
+	{
+		value: 'bur',
+		get label(): string {
+			return __( 'US West' );
+		},
+	},
+	{
+		value: 'dfw',
+		get label(): string {
+			return __( 'US Central' );
+		},
+	},
+	{
+		value: 'dca',
+		get label(): string {
+			return __( 'US East' );
+		},
+	},
+	{
+		value: 'ams',
+		get label(): string {
+			return __( 'EU West' );
+		},
+	},
+];
+
+interface DataCenterFormProps {
+	value: string;
+	onChange: ( value: string ) => void;
+}
+
+export function DataCenterForm( { value, onChange }: DataCenterFormProps ) {
+	const [ shouldShowForm, setShouldShowForm ] = useState( false );
+
+	if ( ! shouldShowForm ) {
+		return (
+			<Text as="p" lineHeight="16px" size={ 12 }>
```

Comment:
Customising the text style here seems a bit off to me. Absent design direction shouldn't we rely on standard styles we get from the components. If we feel we have a case for this sort of custom text style then it feels like we can give it a name and make a component in the `components/` folder.

Reply:
In this case, I just wanted for the text to fit in one line 🙂
I went ahead and removed the styling but shortened the copy, so at least it looks good in English.